### PR TITLE
Add page about Saleor IP Addresses

### PR DESCRIPTION
**Pull request by mmiszy**
Fixes https://github.com/saleor/saleor-docs/issues/1007

### CODE REVIEW

**Review by 2can — APPROVED** [review]

**Review by patrys — APPROVED** [review]

File: docs/developer/extending/webhooks/ip-addresses.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Saleor IP Addresses
+---
+
+## Allowlisting Saleor IP Addresses
+
+Depending on your security requirements, it may be necessary to include Saleor's IP addresses to your firewall's allowlist for receiving webhook events.
```
Comment (timuric) [inline docs/developer/extending/webhooks/ip-addresses.mdx:7]: Depending on your security requirements, including Saleor’s IP addresses to your firewall’s allowlist for receiving webhook events may be necessary.

File: docs/developer/extending/webhooks/ip-addresses.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+To ensure seamless communication with our system, we recommend that you **do not** hardcode a list of Saleor IP addresses in your system. Instead, you must use the following DNS names:
```
Comment (timuric) [inline docs/developer/extending/webhooks/ip-addresses.mdx:9]: I think `reliable` is more accurate than 'seamless' ?

File: docs/developer/extending/webhooks/ip-addresses.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- `out.eu.saleor.cloud` – for the EU region
+- `out.us.saleor.cloud` – for the US region
+
+These DNS names resolve to a list of IP addresses. We strongly advise utilizing domain allowlisting if your firewall supports it. Alternatively, you can resolve the DNS records and cache the IP addresses for a brief duration, typically around 1 hour.
```
Comment (timuric) [inline docs/developer/extending/webhooks/ip-addresses.mdx:14]: `allowlisting` needs emphasis or capitalization

File: docs/developer/extending/webhooks/ip-addresses.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+:::caution
+
+Even when you periodically resolve the DNS records and cache the IP addresses, there's still a potential risk of these addresses changing between cache refreshes. This could lead to your system inadvertently blocking Saleor's webhook events.
```
Comment (timuric) [inline docs/developer/extending/webhooks/ip-addresses.mdx:18]: `inadvertently` can be removed; we should use Basic English

**Review by magul — APPROVED** [review]

**Review by mikolajstempniewicz — APPROVED** [review]

**Review by witoszekdev — APPROVED** [review]

**Pull request merged**